actions/scripts/new.tsx

```diff
@@ -4,18 +4,17 @@ import { SCRIPTS_PATH } from '@/config/env';
 import fs from 'fs/promises';
 
 export async function newFile(name: string, instructions: string, fileName: string) {
-    try {
-        const files = await fs.readdir(SCRIPTS_PATH());
-        const gptFiles = files.filter(file => file.endsWith('.gpt'));
+    const scriptsPath = SCRIPTS_PATH()
+    await fs.mkdir(scriptsPath, {recursive: true})
```

Comment:
Note: `fs.mkdir` doesn't throw an error if the directory already exists when the `recursive: true` option is passed.